app/assets/stylesheets/provider/_authentication_providers.scss

```diff
@@ -22,3 +22,11 @@
 #settings_enforce_sso:not(:checked) + label + .AllowPasswordButton {
   display: block !important;
 }
+
+// HACK: Patternfly 4 uses Font Awesome 5. Icons are now separated in sets (regular, solid, brand)
```

Comment:
I've read this comment 3 times, and still can't understand it.
Well, "nothing is compatible with anything" - I guess that's the summary :sweat_smile: